aws/aws-dynamodb-encryption-java
# chore: Run CodeBuild CI from Github Actions

*Issue #, if available:*

*Description of changes:*

* Delete `buildspec.yml`
* Create `.github/workflows/run-codebuild-ci.yml`. This implements the CodeBuild CI workflow from `buildspec.yml` as a Github actions workflow running CodeBuild steps.

*Testing:*

Successful run: https://github.com/aws/aws-dynamodb-encryption-java/actions/runs/3849450101

There will still be a CodeBuild required check on the PR that will fail, as `buildspec.yml` was removed. I would plan to delete this check from the repo once a reviewer has approved the PR; see release steps below.

*Release steps:*

Once PR is in a state with provisional approval and a failing CodeBuild check:

* Delete CodeBuild required check from repo (should remove check from PR)
* Delete CodeBuild webhook from repo
* Confirm all checks succeed; get approval + merge in

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

# Check any applicable:
- [ ] Were any files moved? Moving files changes their URL, which breaks all hyperlinks to the files.
